domain/litmus_test.go

```diff
@@ -0,0 +1,123 @@
+package domain
+
+import (
+	"testing"
+	"io/ioutil"
+	"github.com/BurntSushi/toml"
+	"github.com/davecgh/go-spew/spew"
+)
+
+func TestUnmarshal(t *testing.T) {
+	b, err := ioutil.ReadFile("testdata/test.toml")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	var tf TestFile
+	if err := toml.Unmarshal(b, &tf); err != nil {
+		t.Fatal(err)
+	}
+
+	spew.Dump(tf)
+}
+
+//func TestEnvFormat(t *testing.T) {
```

Comment:
I remember us commenting this while we were breaking stuff but it'll need to be uncommented (obvs).

Reply:
this whole file needs a complete re-write